card/mod/standard/set/all/rich_html/modal.rb

```diff
@@ -72,7 +72,6 @@ def modal_submit_button opts={}
     modal_link _render_title, size: voo.size
   end
 
-  # @param size [:small, :medium, :large, :full] size of the modal dialog
```

Comment:
maybe we need a comment of this sort?  But there is no `size` param here, and yardoc was complaining.  And I don't know what the right comment would be...